docs/development/conventions/creating-a-release.md

```diff
@@ -58,7 +61,12 @@ This publishes the newly-versioned packages to [npm](https://www.npmjs.com/) and
 $ git push --follow-tags
 ```
 
-Lastly, the `main` branch's latest changes must be merged into the `develop` branch. It's important to do a fast-forward merge to avoid additional merge commits:
+Finally, be sure to follow the [post-release activities](#post-release-activities).
+
+## Post release activities
+
+Following an automated or manual release, the latest changes of `main` must be merged into `develop`.
+It's important to do a fast-forward merge to avoid additional merge commits:
```

Comment:
```suggestion
Following an automated or manual release, the latest changes of `main` must be merged into `develop`. It's important to do a fast-forward merge to avoid additional merge commits:
```